concordium-client.cabal

```diff
@@ -1,10 +1,10 @@
 cabal-version: 2.0
 
--- This file has been generated from package.yaml by hpack version 0.34.4.
+-- This file has been generated from package.yaml by hpack version 0.33.0.
```

Comment:
Please update your stack so that it does not regenerate these files and introduce this diff. Do `stack upgrade`.